infrastructure/ci_iam.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  ci_role_name = trimprefix(data.aws_arn.ci_role.resource, "role/")
+}
+
+data "aws_arn" "ci_role" {
+  arn = local.ci_role_arn
+}
+
+resource "aws_iam_role_policy" "ci_s3_publish" {
+  name   = "content-s3-publish"
+  role   = local.ci_role_name
+  policy = data.aws_iam_policy_document.ci_s3_publish.json
+}
+
+data "aws_iam_policy_document" "ci_s3_publish" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:Get*",
+      "s3:Put*",
+      "s3:Delete*",
+      "s3:ListBucket"
```

Comment:
Why does CI need anything except PutObject?

Reply:
This was from when I was having access policy issues, I will restrict it